client/src/app/components/stardust/Output.tsx

```diff
@@ -279,6 +286,24 @@ class Output extends Component<OutputProps, OutputState> {
         );
     }
 
+    /**
+     * Get output id.
+     * @param type The type to get the output id for.
+     * @returns output id.
+     */
+    private getOutputIdbyType(type: number) {
+        switch (type) {
+            case ALIAS_OUTPUT_TYPE:
+                return this.buildAddressForAliasOrNft();
+            case FOUNDRY_OUTPUT_TYPE:
+                return this.buildFoundyId();
+            case NFT_OUTPUT_TYPE:
+                return this.buildAddressForAliasOrNft();
```

Comment:
Case `ALIAS_OUTPUT_TYPE` and `NFT_OUTPUT_TYPE` can be collapsed in one case, as the call the same funcion.